Mathlib/GroupTheory/QuotientGroup.lean

```diff
@@ -596,6 +597,17 @@ theorem comap_comap_center {H₁ : Subgroup G} [H₁.Normal] {H₂ : Subgroup (G
   simp only [mk'_apply, Subgroup.mem_comap, Subgroup.mem_center_iff, forall_mk, ← mk_mul,
     eq_iff_div_mem, mk_div]
 
+/-- A group is made up of a disjoint union of cosets of a subgroup. -/
+@[to_additive "A additive group is made up of a disjoint union of cosets of an additive
```

Comment:
```suggestion
@[to_additive "An additive group is made up of a disjoint union of cosets of an additive
```

This should go in `Coset.lean`. Or just get the next lemma from `groupEquivQuotientProdSubgroup` and ditch this.